libkbfs/backpressure_disk_limiter.go

```diff
@@ -309,3 +320,62 @@ func (bdl *backpressureDiskLimiter) onBlockDelete(
 	bdl.journalBytes -= blockBytes
 	bdl.updateBytesSemaphoreMaxLocked()
 }
+
+type backpressureDiskLimiterStatus struct {
+	Type string
+
+	BackpressureMinThreshold float64
+	BackpressureMaxThreshold float64
+	MaxJournalByteFrac       float64
+	MaxJournalMB             float64
+	MaxDelaySeconds          float64
+
+	JournalByteFrac     float64
+	DelayScale          float64
+	CurrentDelaySeconds float64
+
+	MBSemaphoreUsageFrac float64
+
+	JournalMB        float64
+	FreeMB           float64
+	MBSemaphoreMax   float64
+	MBSemaphoreCount float64
+}
+
+func (bdl *backpressureDiskLimiter) getStatus() interface{} {
+	bdl.bytesLock.Lock()
+	defer bdl.bytesLock.Unlock()
+
+	journalByteFrac := bdl.calculateJournalByteFrac(
+		bdl.journalBytes, bdl.freeBytes)
+	delayScale := bdl.calculateDelayScale(journalByteFrac)
+	currentDelay := bdl.calculateDelay(context.Background(),
+		bdl.journalBytes, bdl.freeBytes, time.Now())
+
+	MB := float64(1024 * 1024)
+
+	mbSemaphoreMax := float64(bdl.bytesSemaphoreMax) / MB
+	mbSemaphoreCount := float64(bdl.bytesSemaphore.Count()) / MB
+	mbSemaphoreUsageFrac := 1 - mbSemaphoreCount/mbSemaphoreMax
+
+	return backpressureDiskLimiterStatus{
+		Type: "BackpressureDiskLimiter",
+
+		BackpressureMinThreshold: bdl.backpressureMinThreshold,
+		BackpressureMaxThreshold: bdl.backpressureMaxThreshold,
+		MaxJournalByteFrac:       bdl.maxJournalByteFrac,
+		MaxJournalMB:             float64(bdl.maxJournalBytes) / MB,
+		MaxDelaySeconds:          bdl.maxDelay.Seconds(),
+
+		JournalByteFrac:     journalByteFrac,
+		DelayScale:          delayScale,
+		CurrentDelaySeconds: currentDelay.Seconds(),
+
+		MBSemaphoreUsageFrac: mbSemaphoreUsageFrac,
```

Comment:
Done, renamed to `byteUsageFrac`; what I was struggling with was 'byte' is both a descriptor (i.e. as opposed to files) and a unit.